### ISSUE TITLE
Mediatek p22 device (idk if its some or all) can run this shader without bugs

### ISSUE THREAD

**Issue by SevenIsNotNumber** [Open4Es/Open4Es-Shader-Android#31]
:)

**Comment by Adam0530** [comment]
> :)

How? Mine was buggy on Helio g80

**Comment by sermbertti** [comment]
Please fix chip Exynos

**Comment by SavagePy10** [comment]
> > :)
> 
> How? Mine was buggy on Helio g80

Late respond. Try it maybe

**Comment by Creeperrrb** [comment]
What about mediatech mt6580?

**Comment by AjGamingPHOfficial** [comment]
> What about mediatech mt6580?

Try it.
We dont have mediatek device

**Comment by Creeperrrb** [comment]
![Screenshot_20210523-102527](https://user-images.githubusercontent.com/83266075/119248192-597e7f80-bbb1-11eb-982a-4b8583a1fbae.png)
![Screenshot_20210523-102552](https://user-images.githubusercontent.com/83266075/119248197-600cf700-bbb1-11eb-974b-217dd1981bb2.png)
Bruh help me

**Comment by ENDERMANYK** [comment]
Bruh

**Comment by Creeperrrb** [comment]
Can you fix it????
Here is my latest log txt I used another shader after open4es called chocoapic no shadows
https://drive.google.com/file/d/12DkKJX8XWDHTP1Z5w76hkL6D0DThe7Cm/view?usp=drivesdk

**Comment by ENDERMANYK** [comment]
> Can you fix it????
> Here is my latest log txt I used another shader after open4es called chocoapic no shadows
> https://drive.google.com/file/d/12DkKJX8XWDHTP1Z5w76hkL6D0DThe7Cm/view?usp=drivesdk

bruh, don't spam in different Processors issue if you only have one 32bit phone, also update your launcher version and Optifine version

**Comment by Creeperrrb** [comment]
> > Can you fix it????
> > Here is my latest log txt I used another shader after open4es called chocoapic no shadows
> > https://drive.google.com/file/d/12DkKJX8XWDHTP1Z5w76hkL6D0DThe7Cm/view?usp=drivesdk
> 
> bruh, don't spam in different Processors issue if you only have one 32bit phone, also update your launcher version and Optifine version

Bruh I have latest optifine & latest launcher does it effect on opengl version???mine is opengl2.1
& Mediatek mt6580

**Comment by ENDERMANYK** [comment]
> > > Can you fix it????
> > > Here is my latest log txt I used another shader after open4es called chocoapic no shadows
> > > https://drive.google.com/file/d/12DkKJX8XWDHTP1Z5w76hkL6D0DThe7Cm/view?usp=drivesdk
> > 
> > 
> > bruh, don't spam in different Processors issue if you only have one 32bit phone, also update your launcher version and Optifine version
> 
> Bruh I have latest optifine & latest launcher does it effect on opengl version???mine is opengl2.1
> & Mediatek mt6580

It doesn't effect the OpenGL version, if you are using older version optifine, there maybe some GLSL extension your Optifine can't provide so error, launcher can also effect the shader due to it lack LWJGL extension

**Comment by ENDERMANYK** [comment]
> > > Can you fix it????
> > > Here is my latest log txt I used another shader after open4es called chocoapic no shadows
> > > https://drive.google.com/file/d/12DkKJX8XWDHTP1Z5w76hkL6D0DThe7Cm/view?usp=drivesdk
> > 
> > 
> > bruh, don't spam in different Processors issue if you only have one 32bit phone, also update your launcher version and Optifine version
> 
> Bruh I have latest optifine & latest launcher does it effect on opengl version???mine is opengl2.1
> & Mediatek mt6580

Also pls give a screenshot about the shader in the game

**Comment by Creeperrrb** [comment]
> > > > Can you fix it????
> > > > Here is my latest log txt I used another shader after open4es called chocoapic no shadows
> > > > https://drive.google.com/file/d/12DkKJX8XWDHTP1Z5w76hkL6D0DThe7Cm/view?usp=drivesdk
> > > 
> > > 
> > > bruh, don't spam in different Processors issue if you only have one 32bit phone, also update your launcher version and Optifine version
> > 
> > 
> > Bruh I have latest optifine & latest launcher does it effect on opengl version???mine is opengl2.1
> > & Mediatek mt6580
> 
> Also pls give a screenshot about the shader in the game

Just scroll up there is 2 avobe

**Comment by ENDERMANYK** [comment]
> > > > > Can you fix it????
> > > > > Here is my latest log txt I used another shader after open4es called chocoapic no shadows
> > > > > https://drive.google.com/file/d/12DkKJX8XWDHTP1Z5w76hkL6D0DThe7Cm/view?usp=drivesdk
> > > > 
> > > > 
> > > > bruh, don't spam in different Processors issue if you only have one 32bit phone, also update your launcher version and Optifine version
> > > 
> > > 
> > > Bruh I have latest optifine & latest launcher does it effect on opengl version???mine is opengl2.1
> > > & Mediatek mt6580
> > 
> > 
> > Also pls give a screenshot about the shader in the game
> 
> Just scroll up there is 2 avobe

using the latest Optifine still got this?

**Comment by Creeperrrb** [comment]
> > > > > > Can you fix it????
> > > > > > Here is my latest log txt I used another shader after open4es called chocoapic no shadows
> > > > > > https://drive.google.com/file/d/12DkKJX8XWDHTP1Z5w76hkL6D0DThe7Cm/view?usp=drivesdk
> > > > > 
> > > > > 
> > > > > bruh, don't spam in different Processors issue if you only have one 32bit phone, also update your launcher version and Optifine version
> > > > 
> > > > 
> > > > Bruh I have latest optifine & latest launcher does it effect on opengl version???mine is opengl2.1
> > > > & Mediatek mt6580
> > > 
> > > 
> > > Also pls give a screenshot about the shader in the game
> > 
> > 
> > Just scroll up there is 2 avobe
> 
> using the latest Optifine still got this?

Yup lol. But I can't tell my device trash it helped me with lot of things.I guess it isn't made for me. some mcpe shaders also doesn't work for dirlight.

**Comment by Creeperrrb** [comment]
You know what the latest shader open4es beta 1.1
Is working.it is working with bugs but I am getting grass shadow,player shadow, realistic water this is way cooler.also I'm getting 4fps cause I have 2gb ram.BTW cool shader buddy good work

**Comment by Creeperrrb** [comment]
Here are my SS https://drive.google.com/file/d/133WtNescweFc3ohebpK7OfFt3oVy6ksA/view?usp=drivesdk

**Comment by ENDERMANYK** [comment]
> Here are my SS https://drive.google.com/file/d/133WtNescweFc3ohebpK7OfFt3oVy6ksA/view?usp=drivesdk

Can i get the log? I see a invalid program

**Comment by Creeperrrb** [comment]
Here it is https://drive.google.com/file/d/13YQ0wsAHxRuC6ePVl5zS0Z2WqECWh5hg/view?usp=drivesdk

**Comment by Creeperrrb** [comment]
> > Here are my SS https://drive.google.com/file/d/133WtNescweFc3ohebpK7OfFt3oVy6ksA/view?usp=drivesdk
> 
> Can i get the log? I see a invalid program

I don't know why but now all open4es shader works on my device without any bug
O_O & is there any way to fix lag?? Iam getting 4-5fps (mine is 2gb ram & version 1.16.5

**Comment by ENDERMANYK** [comment]
> > > Here are my SS https://drive.google.com/file/d/133WtNescweFc3ohebpK7OfFt3oVy6ksA/view?usp=drivesdk
> > 
> > 
> > Can i get the log? I see a invalid program
> 
> I don't know why but now all open4es shader works on my device without any bug
> O_O & is there any way to fix lag?? Iam getting 4-5fps (mine is 2gb ram & version 1.16.5

I am afraid 2GB is so less for handle the PC game also for the shader..

**Comment by Creeperrrb** [comment]
> > > > Here are my SS https://drive.google.com/file/d/133WtNescweFc3ohebpK7OfFt3oVy6ksA/view?usp=drivesdk
> > > 
> > > 
> > > Can i get the log? I see a invalid program
> > 
> > 
> > I don't know why but now all open4es shader works on my device without any bug
> > O_O & is there any way to fix lag?? Iam getting 4-5fps (mine is 2gb ram & version 1.16.5
> 
> I am afraid 2GB is so less for handle the PC game also for the shader..

I watched a YouTuber running this on 3gb ram still getting 60+ fps so this may not effect also my phone temperature is normal let me try boost fps mods

**Comment by ExoticPlayZYT** [comment]
I have two devices Helio G80 and Helio P22 device on G80 there are black lines and on P22 it runs fine pls solve the issue for G80 because on P22 it works fine

**Comment by DEAEM** [comment]
Disculpen tengo un pequeño error con el sombreado, en si es que aparecen unas rayas negras, el juego me va bien solo es eso de las rallas, habrá alguna solución para eso?, Mi procesador es MediaTek Helio p60 1.99 GHz

**Comment by ExoticPlayZYT** [comment]
Hola, si hay un alma para mediatek helio G70 / G80, estaré encantado
de saberlo, por favor, dígaselo.

**Comment by GearsCloud** [comment]
Lo que me pasa a mí es que le juego me va bien con Shaders pero veo rayas negras, alguna solución? Mi procesador es un MediaTek Helio P70

**Comment by Koijiro** [comment]
> > is there a way to run the shaders without black lines?
can you fix it?

**Comment by Birbs-are-cute** [comment]
@Koijiro buy a new hone withy snapdragon prossecor lmao

**Comment by XMODERLIVESMP** [comment]
How is this an open issue
